geniza/common/management/commands/visual_tests.py

```diff
@@ -40,50 +46,51 @@ def take_snapshots(self, browser):
         browser.find_element_by_css_selector(
             ".doctype-filter li:nth-child(1) label"
         ).click()
-        percy_snapshot(browser, "Document Search filter")
+        percy_snapshot(browser, "Document Search filter%s" % dark_mode_str)
 
         # document search
         browser.get(
             "http://localhost:8000/en/documents/?q=the+writer+Avraham+באנפנא&per_page=2"
         )
-        percy_snapshot(browser, "Document Search")
+        percy_snapshot(browser, "Document Search%s" % dark_mode_str)
 
         # document detail
         browser.get("http://localhost:8000/en/documents/2532/")
-        percy_snapshot(browser, "Document Details")
+        percy_snapshot(browser, "Document Details%s" % dark_mode_str)
 
         # document scholarship
         browser.get("http://localhost:8000/en/documents/9469/scholarship/")
-        percy_snapshot(browser, "Document Scholarship Records")
+        percy_snapshot(browser, "Document Scholarship Records%s" % dark_mode_str)
 
         # mobile menu
         browser.get("http://localhost:8000/en/documents/2532/#menu")
+        mobile_menu_css = "ul#menu { left: 0 !important; transition: none !important; }"
         percy_snapshot(
-            browser,
-            "Mobile menu",
-            percy_css="ul#menu { left: 0 !important; transition: none !important; }",
+            browser, "Mobile menu%s" % dark_mode_str, percy_css=mobile_menu_css
         )
 
         # about submenu open on both desktop and mobile
         browser.get("http://localhost:8000/en/documents/2532/#menu")
         # open about menu
         browser.find_element_by_id("open-about-menu").send_keys(Keys.ENTER)
+        about_menu_css = "ul#about-menu { left: 0 !important; transition: none !important; } @media (min-width: 900px) { ul#about-menu { left: auto !important; } }"
         percy_snapshot(
-            browser,
-            "About submenu",
-            percy_css="ul#about-menu { left: 0 !important; transition: none !important; } @media (min-width: 900px) { ul#about-menu { left: auto !important; } }",
+            browser, "About submenu%s" % dark_mode_str, percy_css=about_menu_css
         )
 
         # 404 page TODO
         # browser.get("http://localhost:8000/bad-url")
-        # percy_snapshot(browser, "404 Page")
+        # percy_snapshot(browser, "404 Page%s" % dark_mode_str)
 
         # 500 page TODO
         # browser.get("http://localhost:8000/500")
-        # percy_snapshot(browser, "500 Page")
+        # percy_snapshot(browser, "500 Page%s" % dark_mode_str)
 
     def handle(self, *args, **options):
         # spin up browser and take snapshots; shut down when finished
         browser = self.get_browser()
+        # light mode
         self.take_snapshots(browser)
+        # dark mode
+        self.take_snapshots(browser, dark_mode=True)
```

Comment:
do we need _every_ snapshot in both dark and light mode? if the layouts and fonts are the same, how much do we gain from having everything available in both? 

@gissoo maybe this is a question for you — do you want/need to review every page in both dark and light mode?